app/src/main/java/io/github/droidkaigi/confsched2017/receiver/NotificationReceiver.java

```diff
@@ -39,7 +37,6 @@ public void onReceive(Context context, Intent intent) {
             Timber.tag(TAG).v("Notification is disabled.");
             return;
         }
-        int sessionId = intent.getIntExtra(KEY_SESSION_ID, 0);
```

Comment:
No problem! Anyway I'm really grad to your contribution! 😃 
But sorry, let me close this PR 🙇 
I'll be so happy if you send another PR.